packages/wdio-cli/src/run.js

```diff
@@ -64,9 +65,7 @@ export default function run (params) {
     })
 }
 
-function launch (wdioConf, params) {
+export function launch (wdioConf, params) {
     const launcher = new Launcher(wdioConf, params)
-    launcher.run().then(
-        (code) => process.nextTick(() => process.exit(code)),
-        (e) => process.nextTick(() => { throw e }))
+    return launcher.run().catch(err => { console.error(err) })
```

Comment:
Why do we log the error instead of letting the user deal with it?

Reply:
sorry, missed that after debugging, removed.